<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSplashScreen</class>
 <widget class="QWidget" name="GSplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>811</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLabel" name="Startlabel_1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>360</width>
       <height>340</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/bg/b.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从此告别拖延和忘记！保持高效和清晰的头脑。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Slabel_title_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>491</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>欢迎使用！</string>
     </property>
    </widget>
    <widget class="QLabel" name="Slabel_title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>491</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QTDO待办专家</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="Startlabel_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>360</width>
       <height>340</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/bg/a.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Slabel_title_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>让你轻松管理</string>
     </property>
    </widget>
    <widget class="QLabel" name="Slabel_title_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>高效目标</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;创建自己的事项清单，安排好每一天的价值！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="Startlabel_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>360</width>
       <height>340</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/bg/c.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Slabel_title_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>护你便捷贴心</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;让您的生活更加有条不紊！让您的工作效率翻倍。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Slabel_title_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>到点提醒</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="INkown">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>350</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>不等待，立即前往</string>
   </property>
   <property name="text">
    <string>      前往 &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_logo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>GTDO待办专家</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/logo.png</normaloff>:/image/logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>354</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>120</y>
     <width>480</width>
     <height>280</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/bg/wbg.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="INkown_close">
   <property name="geometry">
    <rect>
     <x>678</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>取消等待，退出程序</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/del.png</normaloff>:/image/del.png</iconset>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>INkown</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>Start_logo</zorder>
  <zorder>label_time</zorder>
  <zorder>INkown_close</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
